### PR TITLE
fix(agw): Readme Update and Minor fixes

### PR DESCRIPTION
**Pull request by Alt-Shivam**
Signed-off-by: Alt-Shivam <shivank1128@gmail.com>

## Summary
Agw Helm charts are currently in experimental state, therefore to get them in working state,
This PR adds some extra information to Readme.md, and fixes minor issues to Helm charts.

### CODE REVIEW

**Comment by viprat** [comment]
Helm Deployment of AGW

File: lte/gateway/deploy/agwc-helm-charts/README.md [hunk 1]
```diff
@@ -16,6 +16,32 @@ The following table list the configurable parameters of the agw chart and their
 | `image.tag` | Tag for agwc image. | `latest` |
 | `config.domainName` | Orchestrator domain name. | `` |
 | `persistant.name` | Secret name containing agwc certs. | `agwc-claim` |
+| `config.gwChallenge` | Challenge key. | `` |
```
Comment (ShubhamTatvamasi) [inline lte/gateway/deploy/agwc-helm-charts/README.md:20]: The `gwChallenge` will get created automatically, or we have to create is manually first?

Comment (ShubhamTatvamasi) [reply lte/gateway/deploy/agwc-helm-charts/README.md:20]: I see in the docs you have mentioned that we have to create `rootCA.pem`, `snowflake` and `gw_challenge.pem` manually first. Is that right?

Comment (ShubhamTatvamasi) [reply lte/gateway/deploy/agwc-helm-charts/README.md:20]: Can't we generate this automatically using a Kubernetes job and then deploy the helm chart?

Comment (Alt-Shivam) [reply lte/gateway/deploy/agwc-helm-charts/README.md:20]: we've to manually create `gw_challenge.pem` and `snowflake` first.
`rootCA.pem` from orc8r deployment.
and sure, we can have a K8s job to automate this.
will add a job soon.
Thanks

Comment (ShubhamTatvamasi) [reply lte/gateway/deploy/agwc-helm-charts/README.md:20]: Thanks @Alt-Shivam

**Review by ajahl — APPROVED** [review]

**Comment by voisey** [comment]
Could you please rebase this to get the CI to run again @Alt-Shivam?

**Comment by mpfirrmann** [comment]
What is the status of this PR? Can this be merged? @Alt-Shivam @ShubhamTatvamasi

**Comment by panyogesh** [comment]
@Alt-Shivam : Can you please rebase and will try to get it reviewed and merged.

**Comment by Alt-Shivam** [comment]
Thanks @panyogesh @mpfirrmann @voisey for drawing my attention at this PR 👀,
I think it can be merged now.

**Review by jordanvrtanoski — APPROVED** [review]

**Pull request merged**